src/backend/core/migrations/0002_team_slug.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 5.0.1 on 2024-02-01 14:24
```

Comment:
Until we go to production, I propose to modify 0001_initial instead of adding migrations.
adding migrations when we still break things a lot would mean writing data migrations which we don't want to do until we have to! 
maybe add a warning in the README that this project is not yet for production and that we will still break things like the database schema?

Reply:
Gotcha, I'll modify 0001_initial and add a note on README.